### ISSUE TITLE
nasa_ads query returns 'None' for citations counts

### ISSUE THREAD

**Issue by fpavogt** [astropy/astroquery#1336]
In astroquery v0.3.9 and v0.3.10 (pip installed), nasa_ads queries fail to return the number of citations associated with a given entry. 

Here's a MWE: 

```
from astroquery import nasa_ads as ads
query = '2018ascl.soft06027V'
result = ads.ADS.query_simple(query, cache = False)
print(result['citations'])
```
In v0.3.9 and 0.3.10, this returns:
```
citations [1]
-------------
         None
```
In v0.3.9, this returns:
```
citations [1]
-------------
3
```
Running Python 3.6.8 via a terminal shell with ipython 6.1.0 on Mac OSX 10.13.6. 
Any idea on what may be going on here ?

**Comment by keflavich** [comment]
This is a bug in the configured fields; `citations` should be `citation`

**Comment by fpavogt** [comment]
Hum ... I think it should actually be `citation_count`, as per [the ads api doc](https://github.com/adsabs/adsabs-dev-api/blob/master/Search_API.ipynb). 

Just tried to swap `citations` for `citation_count` on l.33 in __init__.py of the nasa_ads submodules, and it works fine.

**Comment by keflavich** [comment]
We probably want both `citation` and `citation_count` by default.

**Comment by keflavich** [comment]
See #1337

**Comment by fpavogt** [comment]
Excellent!

**Comment by bsipocz** [comment]
Fixed by #1337
